addons/l10n_in_edi_ewaybill/models/ewaybill_type.py

```diff
@@ -30,3 +31,11 @@ def name_get(self):
             name += _(" (Sub-Type: %s)", ewaybill_type.sub_type)
             result_dict[ewaybill_type.id] = name
         return list(result_dict.items())
+
+    @api.model
+    def _search(self, domain, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
+        context = self._context or {}
+        search_mode = context.get('l10n_in_type_search_mode')
+        if isinstance(search_mode, str) and search_mode in ('in', 'out'):
```

Comment:
does it need the `isinstance`?
`if search_mode in ('in', out')` seems ok to me

Reply:
Not sure what the l10n_in_type_search_move is good for.